rust/scx_utils/src/builder.rs

```diff
@@ -33,8 +37,37 @@ impl Builder {
         }
     }
 
+    fn find_vmlinux_h<P: AsRef<Path> + AsRef<OsStr>>(
+        dest: P,
+        kernel_target: &str,
+    ) -> Result<String> {
+        let entries = fs::read_dir(&dest)?;
+
+        for entry in entries {
+            let entry = entry?;
+            let file_name = entry.file_name();
+            let file_name_str = file_name.to_string_lossy();
+
+            println!("{file_name_str}");
+            if file_name_str.contains(&kernel_target) {
+                return Ok(entry.path().to_string_lossy().into_owned());
+            }
+        }
+        Err(anyhow!("vmlinux.h for arch {} is not found", kernel_target))
+    }
+
     fn gen_bindings(&self) {
         let out_dir = env::var("OUT_DIR").unwrap();
+        let clang = ClangInfo::new().unwrap();
+        let vmlinux_dir = Path::new(&BPF_H)
```

Comment:
I think I like this (and all signals green). 

@hodgesds what do you think (roughly having build.rs create the appropriate symlink for each platform). 

It might make some editor stuff work less well but like, worst case anyone bothered by that could just symlink their platform's `.h` to the right one in their src dir and this solves a real problem sooooo idk?

Reply:
> If we go w/ this, I think I'd try this out: https://github.com/arighi/virtme-ng/releases/tag/v1.30 . In theory, it shouldn't be that tricky expanding out test suite to cover arm.

I've uploaded the new virtme-ng version on pypi, so I think we're already using v1.30 in our CI.